indra/newview/llviewerregion.cpp

```diff
@@ -3721,6 +3721,10 @@ bool LLViewerRegion::bakesOnMeshEnabled() const
 
 bool LLViewerRegion::meshRezEnabled() const
 {
+    if(!mSimulatorFeaturesReceived)
```

Comment:
Suggestion: add an in-code comment that explains why this hack is needed. 

P.S. By this point meshRezEnabled is mission critical. Perhaps just deprecate it and assume meshes are always supported?